From 4eb9db734bcf06d293415160053791b0ee4844bf Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 30 Apr 2015 13:47:17 +0800
Subject: [PATCH] MLK-10783-6 doc: dt-binding: usbmisc_imx: add compatible
 string "fsl,imx6ul-usbmisc"

Add compatible string "fsl,imx6ul-usbmisc"

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 Documentation/devicetree/bindings/usb/usbmisc-imx.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/usb/usbmisc-imx.txt b/Documentation/devicetree/bindings/usb/usbmisc-imx.txt
index f1e27fa..735ed9d 100644
--- a/Documentation/devicetree/bindings/usb/usbmisc-imx.txt
+++ b/Documentation/devicetree/bindings/usb/usbmisc-imx.txt
@@ -7,6 +7,7 @@ Required properties:
 	"fsl,vf610-usbmisc" for Vybrid vf610
 	"fsl,imx6sx-usbmisc" for imx6sx
 	"fsl,imx7d-usbmisc" for imx7d
+	"fsl,imx6ul-usbmisc" for imx6ul
 - reg: Should contain registers location and length
 
 Examples:
-- 
1.8.0

